### DISCUSSION TITLE
0.2.2 - Easy Launcher

### DISCUSSION THREAD

**Discussion by CreativeCodeCat** [DroidWorksStudio/EasyLauncher#109] [Check for Updates]
# What's Changed

### Bug Fixes:

- Fixed the App update helper - ([57a680d](https://github.com/DroidWorksStudio/EasyLauncher/commit/57a680d96befddb38fccc8d7e1078e8d9382a08c))
- Android 15: Fixed the set default launcher. - ([3fd3dc4](https://github.com/DroidWorksStudio/EasyLauncher/commit/3fd3dc49d9a9a28a4eff5597d027b323fb3a11f6))

### Releases:

- Prepare for version 0.2.2 - ([0a28ef7](https://github.com/DroidWorksStudio/EasyLauncher/commit/0a28ef74485fb5cd64a7519c4b74dfe8b5f327f2))

**Full Changelog**: https://github.com/DroidWorksStudio/EasyLauncher/compare/0.2.1...0.2.2

<hr /><em>This discussion was created from the release <a href='https://github.com/DroidWorksStudio/EasyLauncher/releases/tag/0.2.2'>0.2.2 - Easy Launcher</a>.</em>

**Comment by Nakul-shah1** [comment]
Please remove the favourite app from home screen as another page is dedicated to them.

**Reply by CreativeCodeCat** [reply]
Why would I do that when the home screen is where they are displayed on, the favourite screen is for if you wanna reorder them.

So this isn't going to happen sorry

**Comment by Nakul-shah1** [comment]
Hi hecodes2much,
You have worked really hard on both the Launchers. This is really awesome. Keep working hard

**Comment by Nakul-shah1** [comment]
There's a really good launcher known as Kiss launcher you can take some reference from it.
